todo_app/templates/index.html

```diff
@@ -9,38 +9,66 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>To Do Items</h2>
 
-      <ul class="list-group mb-4">
-        {% for item in items %}
-          <li>{{ item.name }} -- {{ item.desc }} -- {{ item.listName }}</li>
+      <ul class="list-group mb-4" id="todoItems">
+        {% for item in view_model.todo_items %}
+          <li>{{ item.name }} -- {{ item.desc }}</li>
           
-          {% if item.listName == 'To Do' %}
-            <form id="completeItem{{item.id}}" method="POST" action="/start/{{item.id}}">
-              <input type="submit" value="Start" />
-            </form>
-          {% else %}
-            <form id="completeItem{{item.id}}" method="POST" action="/reset/{{item.id}}">
-              <input type="submit" value="Move back to todo" />
-            </form>
-          {% endif %}
-
-          {% if item.listName != 'Done' %}
-            <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}">
-              <input type="submit" value="Complete" />
-            </form>
-          {% endif %}
-
-
-          <a onclick="deleteItem('{{ item.id }}')" href="#">Delete</a>
+          <form id="completeItem{{item.id}}" method="POST" action="/start/{{item.id}}">
+            <input type="submit" value="Start" />
+          </form>
+        
+          <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}">
+            <input type="submit" value="Complete" />
+          </form>
+
+          <a id="delete{{item.id}}" onclick="deleteItem('{{ item.id }}')" href="#">Delete</a>
+        {% endfor %}
+      </ul>
+
+
+      <h2>Doing Items</h2>
+
+      <ul class="list-group mb-4" id="doingItems">
+        {% for item in view_model.doing_items %}
+          <li>{{ item.name }} -- {{ item.desc }}</li>
+          
+          <form id="completeItem{{item.id}}" method="POST" action="/reset/{{item.id}}">
+            <input type="submit" value="Move back to todo" />
+          </form>
+
+          <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}">
+            <input type="submit" value="Complete" />
```

Comment:
These two forms have the same id `completeItem{{item.id}}` which is wrong.

But also, why do these forms need an id?